modules/apps/static/portal-configuration/portal-configuration-metatype-api/build.gradle

```diff
@@ -4,7 +4,11 @@ dependencies {
 	compileOnly group: "org.osgi", name: "org.osgi.core", version: "5.0.0"
 	compileOnly group: "org.osgi", name: "org.osgi.service.component.annotations", version: "1.3.0"
 	compileOnly group: "org.osgi", name: "org.osgi.service.metatype", version: "1.3.0"
-	compileOnly group: "org.ow2.asm", name: "asm-debug-all", version: "5.0.4"
+	compileOnly group: "org.ow2.asm", name: "asm", version: "7.0"
```

Comment:
@dantewang 
You changed three build.gradle files, could you please check if we need to include every jar in each of them?

This should have nothing to do with the export package from system.packages.extra.bnd.

Thanks.
Tina.